src/main/java/org/broadinstitute/hellbender/tools/recalibration/BQSRGatherer.java

```diff
@@ -1,84 +0,0 @@
-package org.broadinstitute.hellbender.tools.recalibration;
-
-import org.apache.commons.collections4.CollectionUtils;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
-import org.broadinstitute.hellbender.exceptions.GATKException;
-import org.broadinstitute.hellbender.utils.commandline.Gatherer;
-import org.broadinstitute.hellbender.utils.io.IOUtils;
-import org.broadinstitute.hellbender.utils.report.GATKReport;
-
-import java.io.File;
-import java.io.IOException;
-import java.io.PrintStream;
-import java.util.*;
-
-/**
- * Merges BQSR Recaliration tables.
- */
-public final class BQSRGatherer extends Gatherer {
```

Comment:
I'm not sure that we can remove this class. It's used by the non-Dataflow version of BaseRecalibrator. It's done in a magic way, via annotation, but removing this may break the non-cloud version.

So removing this is a big step. If that's what we mean to do then it should at a minimum be mentioned in the PR description.